torchgeo/datasets/chesapeake.py

```diff
@@ -618,7 +618,9 @@ def __init__(
                             "prior_from_cooccurrences_101_31_no_osm_no_buildings": prior_fn,  # noqa: E501
                         },
                     )
-
+    @property
+    def _nlcd_cmap(self):
```

Comment:
Why is the property needed?

Reply:
Provides encapsulation and improving code readability for future changes in how colormap is derived.

Reply:
We already have that with the `self.cmap` attribute, I don't think we need a second attribute.

Reply:
I agree